### PR TITLE
Add bsddb3

### LINKED ISSUE #4493: Package bsddb3
Would be good to package [`bsddb3`]( https://pypi.python.org/pypi/bsddb3 ). This requires `libdb`. ( https://github.com/conda-forge/staged-recipes/issues/4492 )

### PR DESCRIPTION
**Pull request by shadowwalkersb**
Closes #4493.

### CODE REVIEW

**Comment by shadowwalkersb** [comment]
@conda-forge/staged-recipes, ready for review.

**Comment by scopatz** [comment]
Thanks @shadowwalkersb!

**Comment by shadowwalkersb** [comment]
I wasn't sure about the licensing. I suppose it is OK, but it would be nice to confirm.

Also, I am not sure if I did the build matrix handling correctly. It doesn't have to be `noarch: python`?

**Comment by scopatz** [comment]
I checked the license and you are fine here. Does not need to be noarch

File: recipes/bsddb3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  skip: True     # [win or py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+    - libdb
+  run:
+    - python
+    - libdb
```
Comment (isuruf) [inline recipes/bsddb3/meta.yaml]: You need to either add pinning here or add run_exports to libdb

**Pull request merged**